pkg/controller.v1alpha3/suggestion/suggestion_controller_util.go

```diff
@@ -0,0 +1,36 @@
+package suggestion
+
+import (
+	"context"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileSuggestion) createOrUpdateDeployment(deploy *appsv1.Deployment) (*appsv1.Deployment, error) {
```

Comment:
Do we need to update the deployment if already present?

Reply:
No idea about if we have such a use case now.

Reply:
Ok. Function name is confusing. Either we can name to ReconcileDeployment?

Reply:
Sure